pkg/cloudprovider/providers/azure/azure_zones.go

```diff
@@ -18,14 +18,15 @@ package azure
 
 import (
 	"encoding/json"
-	"errors"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"sync"
 
+	"github.com/Azure/azure-sdk-for-go/arm/compute"
```

Comment:
Can we group the `github.com/Azure` imports separate from the `k8s.io` imports and group `strconv` with the rest of the stdlib packges?

Reply:
please fix

Reply:
yup sure